code/modules/surgery/surgery.dm

```diff
@@ -10,7 +10,7 @@
 	var/list/possible_locs = list() 							//Multiple locations -- c0
 	var/ignore_clothes = 0										//This surgery ignores clothes
 	var/obj/item/organ/organ									//Operable body part
-
+	var/requires_missing_bodypart = 0							//Surgery available only when a bodypart is present, or only when it is missing.

```

Comment:
To be honest, changing it to "requires_bodypart" would probably be better. Less double negatives. If there were agnostic surgeries, we could get away with adding another var "requires_absence", but that just means that you could theoretically get the impossible case of a surgery both requiring the presence and absence of a limb.